Core/ApplicationManagement/Services/CategoryService/CategoryService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Core.Common.CreateViewModels;
+using Core.Common.ViewModels;
+using DataAccess.Entities;
+using DataAccess.Infrastructure.UnitOfWork;
+
+namespace Core.ApplicationManagement.Services.CategoryService
+{
+    public class CategoryService : ICategoryService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+
+        public CategoryService(IUnitOfWork unitOfWork)
+        {
+            _unitOfWork = unitOfWork;
+        }
+
+        public async Task Create(CreateCategoryViewModel viewModel)
```

Comment:
тут моделька выглядит избыточной, можно просто строку передавать
```suggestion
        public async Task Create(string name)
```